.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] There is no functional readme file.  Go to this [repo](https://github.com/microverseinc/readme-template), grab the README.md template there, and customize it for your project.

Remember, your readme needs to have specific instructions about how to setup and run your project, as well as your tests.  👍